cypress/e2e/tests/pages/global-settings/settings-p2.spec.ts

```diff
@@ -82,7 +82,7 @@ describe('Settings', { testIsolation: 'off' }, () => {
     settingsPage.settingsValue('system-default-registry').contains(settings['system-default-registry'].new);
 
     // Check cluster manager > create
-    const createRKE2ClusterPage = new ClusterManagerCreateRke2CustomPagePo();
+    const createRKE2ClusterPage = new ClusterManagerCreateRke2CustomPagePo('local');
```

Comment:
Think this is probably the time to start using the `BLANK_CLUSTER` id (`_`) which is what the dashboard uses for all cluster management pages (and others). It's probably best to set it as the default in the POs

The `local` cluster id was how it used to work but not anymore.

Reply:
I've only updated the parent class (`ClusterManagerCreatePagePo`) to have the default since all the other POs extend it and would inherit the default value. But let me know if you would still like me to set the default in the other POs

Reply:
The aim is to convert the old references from `local` to the `_` value when using cluster manager based pages and not add any new ones. 

Although the usages below aren't really new ones, they do prompt others to copy and paste with the bad value. What happens if `local` is removed from the files below and a default `_` is used instead?

cypress/e2e/tests/pages/manager/agent-configuration-rke2.spec.ts
cypress/e2e/tests/pages/manager/cilium-cni.spec.ts
cypress/e2e/tests/pages/manager/cluster-manager.spec.ts
cypress/e2e/tests/pages/manager/node-templates.spec.ts
cypress/e2e/tests/pages/manager/registries.spec.ts
cypress/e2e/tests/pages/manager/rke-templates.spec.ts

Reply:
sorry i misunderstood, makes sense now. After removing 'local' from the class references from the our tests now use the default '_'. I've updated all of the tests under `cypress/e2e/tests/pages/manager` in this way.

Reply:
Global Settings doesn't reset the target cluster, so should retain whatever it had previously. Cluster Management --> Global Settings `_` and Cluster Explorer --> Global Settings `<cluster id>`. It should be the same also from Home page --> Global Settings, going from blank cluster to blank cluster, but it looks like it uses some cached version of the previous cluster id (even on refresh). So there's definitely something wrong dashboard code side.

You've gone over an above on this one though. No new locals and a lot of locals converted. Good job.